### DISCUSSION TITLE
Looking for a Changelog

### DISCUSSION THREAD

**Discussion by kryban93** [GetStream/stream-chat-angular#535] [General]
Hello, 
I'm looking for a Stream-chat-angular changelog. I can easily find changelog for Stream-chat, but is there any specifically for stream-chat-angular?

Greetings!

**Comment by szuperaz** [comment]
Hi, you can find that here: https://github.com/GetStream/stream-chat-angular/releases

**Reply by kryban93** [reply]
Thanks for the answer

**Reply by szuperaz** [reply]
Sure, feel free to ask us if you have any more questions.
